### DISCUSSION TITLE
OAuth dependencies outdated

### DISCUSSION THREAD

**Discussion by LindezaGrey** [fastapi-users/fastapi-users#1119] [Q&A] [answered]
## Describe the bug

I want to use a generic OpenID-Connect provider as outlined here https://frankie567.github.io/httpx-oauth/oauth2/#openid
this feature was added in v.0.10.0

fastapi-users pins the optional dependency to <0.8
https://github.com/fastapi-users/fastapi-users/blob/baca17d9e029aafb757e128cedbb6782e45bcd89/pyproject.toml#L144

Is there a specific reason this limitation exists, or could it be bumped?

Raw output from poetry:

```
Using version ^0.10.1 for httpx-oauth

Updating dependencies
Resolving dependencies... (0.0s)

  SolverProblemError

  Because fastapi-users (10.2.0) depends on httpx-oauth (>=0.4,<0.8)
   and no versions of fastapi-users match >10.2.0,<11.0.0, fastapi-users (>=10.2.0,<11.0.0) requires httpx-oauth (>=0.4,<0.8).
  So, because my-project depends on both fastapi-users (^10.2.0) and httpx-oauth (^0.10.1), version solving failed.
```

EDIT: 
I just saw, there is already a dependabot pull request:

https://github.com/fastapi-users/fastapi-users/pull/1106

**Comment by frankie567** [comment, marked as answer]
Indeed! There is no particular reason with this, I'll make a release with the updated dependencies.

**Reply by frankie567** [reply]
Released in [v10.2.1](https://github.com/fastapi-users/fastapi-users/releases/tag/v10.2.1)!

**Reply by LindezaGrey** [reply]
oh wow , that was fast! very much appreciated
